src/app/components/button/button.test.js

```diff
@@ -28,12 +27,6 @@ test('renders correctly with primary variant', () => {
   expect(asFragment()).toMatchSnapshot();
 });
 
-// test('has the correct class when variant is primary', () => {
```

Comment:
Question: why are we commenting out a test in another area of the code base?

Reply:
That test was already commented out from my last merge request. I removed it since we have the snap shot test in there